### DISCUSSION TITLE
Which human spacecraft uses CFS?

### DISCUSSION THREAD

**Discussion by ephraim271** [nasa/cFS#728] [Q&A] [answered]
From description : 
![image](https://github.com/nasa/cFS/assets/120187203/e3728e41-7357-4fd2-a31b-7926ef1bd26a)

**Comment by skliper** [comment, marked as answer]
https://ntrs.nasa.gov/api/citations/20190000011/downloads/20190000011.pdf

**Reply by skliper** [reply]
https://www.nasa.gov/solar-system/goddards-core-flight-software-chosen-for-nasas-lunar-gateway/

**Reply by ephraim271** [reply]
Oh I'm so glad something like this is open source!
